x/clock/keeper/querier.go

```diff
@@ -24,13 +24,48 @@ func NewQuerier(k Keeper) Querier {
 func (q Querier) ClockContracts(stdCtx context.Context, _ *types.QueryClockContracts) (*types.QueryClockContractsResponse, error) {
 	ctx := sdk.UnwrapSDKContext(stdCtx)
 
-	p := q.keeper.GetParams(ctx)
+	contracts := q.keeper.GetAllContracts(ctx, false)
```

Comment:
this likely needs to be paginated sadly. same for jailed

Reply:
Added pagination